packages/slate/src/models/range.js

```diff
@@ -129,11 +130,14 @@ class Range extends Record(DEFAULTS) {
       isAtomic = false,
     } = object
 
+    const {
```

Comment:
Here we need to accept `anchorKey/Offset` style old names, but log a deprecation warning when they are used. And instead move to using the newer `anchor.key/offset` objects.